addons/l10n_syscohada/__manifest__.py

```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-# Copyright (C) 2010-2011 BAAMTU SARL (<http://www.baamtu.sn>).
-# contact: leadsn@baamtu.com
 
 {
-    'name' : 'OHADA - Accounting',
-    'author' : 'Baamtu Senegal',
+    'name': 'OHADA - Accounting',
+    'author': 'Odoo SA',
```

Comment:
I don't know if we want to remove their name, but if we do, then we also need to change the website

Reply:
Btw, I would say 'Odoo S.A.' and not 'Odoo SA' is the default, as seen here https://github.com/odoo/odoo/blob/master/odoo/modules/module.py#L30

Reply:
Will change that asap then